backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardQueryService.kt

```diff
@@ -5,23 +5,26 @@ import app.ehrenamtskarte.backend.exception.service.ProjectNotFoundException
 import app.ehrenamtskarte.backend.verification.database.CodeType
 import app.ehrenamtskarte.backend.verification.service.CardVerifier
 import app.ehrenamtskarte.backend.verification.webservice.schema.types.CardVerificationModel
+import app.ehrenamtskarte.backend.verification.webservice.schema.types.CardVerificationResultModel
 import com.expediagroup.graphql.generator.annotations.GraphQLDescription
 import graphql.schema.DataFetchingEnvironment
+import java.time.Instant
 import java.util.Base64
 
 @Suppress("unused")
 class CardQueryService {
-    @GraphQLDescription("Returns whether there is a card in the given project with that hash registered for that this TOTP is currently valid")
-    fun verifyCardInProject(project: String, card: CardVerificationModel, dfe: DataFetchingEnvironment): Boolean {
+    @GraphQLDescription("Returns whether there is a card in the given project with that hash registered for that this TOTP is currently valid and a timestamp of the last check")
+    fun verifyCardInProject(project: String, card: CardVerificationModel, dfe: DataFetchingEnvironment): CardVerificationResultModel {
```

Comment:
Hm okay and you think its necessary to get the timestamp from the backend instead of using the `DateTime` from the client app. In any case we can not avoid manipulation of the system time. So we could also use the client side timestamp

Reply:
I think it would be slightly better to use a timestamp created in the backend: Then whenever the app *does* have internet connection, the timestamp is updated, and the user would have to have set his system date/time to the backend time. It is a bit harder to workaround.
I'm not sure, if it's worth the increased complexity though. You have to decide :D